pubsub/gcloud/aio/pubsub/subscriber_client.py

```diff
@@ -17,9 +17,13 @@ class SubscriberClient:
 
     def __init__(self,
                  *,
+                 credentials: Optional = None,
                  loop: Optional[asyncio.AbstractEventLoop] = None
                  ) -> None:
-        self._subscriber = pubsub.SubscriberClient()
+        if credentials:
+            self._subscriber = pubsub.SubscriberClient(credentials=credentials)
```

Comment:
Looks like this client [actually accepts kwargs](https://github.com/googleapis/google-cloud-python/blob/b4ea85cd6de240956b44ed25a4d449a9c3f1649d/pubsub/google/cloud/pubsub_v1/subscriber/client.py#L73). Rather than whitelisting the args we're aware of, might be easiest to just pass through everything -- it'd be nice to be able to avoid having to make a PR like this for each new feature that we aren't yet exposing.